Jenkinsfile

```diff
@@ -35,5 +35,18 @@ pipeline {
                 sh 'tfdescsan --test --tsv https://dcos-terraform-mappings.mesosphere.com/ --var outputs.tf --cloud "$(echo ${JOB_NAME##*/terraform-} | sed -E "s/(rm)?-.*//")"'
             }
         }
+        stage('Integration Test') {
+            agent { label "dcos-terraform-cicd" }
+            steps {
+                withCredentials([
+                   [$class: 'AmazonWebServicesCredentialsBinding', credentialsId: 'dcos-terraform-ci-aws'],
+                   azureServicePrincipal('dcos-terraform-ci-azure'),
+                   [$class: 'FileBinding', credentialsId: 'dcos-terraform-ci-gcp', variable: 'GOOGLE_APPLICATION_CREDENTIALS']
+                   ]) {
+                         sh 'printenv'
```

Comment:
please remove. There is no need showing the environment.

Reply:
yep! Sounds good!